@3rdweb-sdk/react/hooks/useGas.ts

```diff
@@ -0,0 +1,15 @@
+import { useQuery } from "react-query";
+import { dashboardKeys } from "..";
+
+export function useGas() {
+  return useQuery(
+    dashboardKeys.gas(),
```

Comment:
just make this an array

```suggestion
    ["gas-price", "ethereum"]
```